### ISSUE TITLE
crash after dbus_message_unref

### ISSUE THREAD

**Issue by minlexx** [minlexx/wmsystemtray-plasma5#1]
openSUSE 42.1 beta running in a vm
app log:
```
> ./wmsystemtray             (null): Parsing args
wmsystemtray: Creating windows
wmsystemtray: My pid is 2036
wmsystemtray: Opening display ':0'
wmsystemtray: Interning atoms
wmsystemtray:   _NET_WM_PING = 274
wmsystemtray:   _NET_WM_PID = 313
wmsystemtray:   WM_PROTOCOLS = 271
wmsystemtray:   WM_DELETE_WINDOW = 272
wmsystemtray: Loading XPM
wmsystemtray: Allocating window hints
wmsystemtray: Parsing geometry string ''
wmsystemtray: Result geometry: 0 0 64 64
wmsystemtray: Got hostname: [dyn2]
wmsystemtray: Allocating space for 1 windows
wmsystemtray: Creating selection window
wmsystemtray: Creating GCs
wmsystemtray: Dock window #0 is 4e00009
wmsystemtray: Freeing X hints
wmsystemtray: Updating display: page 1 of 1
wmsystemtray: Drawing window 0 (4e00009)
wmsystemtray: Initializing protocols
wmsystemtray: fdtray: Loading atoms
wmsystemtray: fdtray: Grabbing system tray selection
wmsystemtray: X time is 108302
wmsystemtray: fdtray: Notifying clients that we are now the system tray
wmsystemtray: DBus: version: 1.8.16
wmsystemtray:   DBus: SIGNAL NameAcquired(): [:1.50]
process 2036: arguments to dbus_message_unref() were incorrect, assertion "!message->in_cache" failed in file dbus-message.c line 1696.
This is normally a bug in some application using the D-Bus library.
  D-Bus not built with -rdynamic so unable to print a backtrace
Aborted
```

**Comment by minlexx** [comment]
Again, one more time
```
msystemtray: ===== Entering main loop =====
wmsystemtray: fdtray: Dock request for window 4e016b7
wmsystemtray: Adding tray icon 4e016b7 of type 0
wmsystemtray: DBus: SNI: will try to register new SNI with name [org.freedesktop.StatusNotifierItem-2589-1]
wmsystemtray: DBus: SNI: will use icon/tooltip title [Thunderbird]
wmsystemtray: DBus: SNI: Request name [org.freedesktop.StatusNotifierItem-2589-1] OK!
wmsystemtray:   DBus: SIGNAL NameAcquired(): [org.freedesktop.StatusNotifierItem-2589-1]
wmsystemtray: Mapping tray icon 4e016b7
wmsystemtray: Updating display: page 1 of 1
wmsystemtray: Drawing window 0 (4c00008)
wmsystemtray:   Drawing tray icon
wmsystemtray:   [0] Tray icon 4e016b7 at 0 (8,4)
wmsystemtray:     Reparenting 4e016b7 to 4c00008
wmsystemtray:   DBus: attrs for window: [24x24], map_state=1
wmsystemtray:   DBus: X11: ignoring BadMatch error
wmsystemtray:   DBus: X11: ERROR: failed to get window image, window 4e016b7
wmsystemtray: X time is 539261
wmsystemtray: Updating display: page 1 of 1
wmsystemtray: Drawing window 0 (4c00008)
wmsystemtray:   Drawing tray icon
wmsystemtray:   [0] Tray icon 4e016b7 at 0 (8,4)
wmsystemtray:   DBus: attrs for window: [24x24], map_state=2
wmsystemtray:   DBus: SNI: got image (24 x 24), depth=24, byte_order=0, bytes_per_line=96, bits_per_pixel=32, bitmap_unit=32, bytes_per_line=96
wmsystemtray:     Masks: R ff0000 G ff00 B ff
wmsystemtray: DBus: SNI: signal NewIcon() sent OK.
wmsystemtray: DBus: SNI: Properties.GetAll( "org.kde.StatusNotifierItem" )
process 2589: arguments to dbus_message_unref() were incorrect, assertion "!message->in_cache" failed in file dbus-message.c line 1696.
This is normally a bug in some application using the D-Bus library.
  D-Bus not built with -rdynamic so unable to print a backtrace
Aborted
```

**Comment by minlexx** [comment]
Should be fixed after commit 5f0e4f04b84a9bf44a2c9971a4b3fba38efc1c96
